templates/app/plugins/README.md

```diff
@@ -1,16 +0,0 @@
-# Plugins Folder
-
-Plugins define behavior that is common to all the routes in your
-application. Authentication, caching, templates, and all the other cross
-cutting concerns should be handled by plugins placed in this folder.
-
-Files in this folder are typically defined through the
-[`fastify-plugin`](https://github.com/fastify/fastify-plugin) module,
-making them non-encapsulated. They can define decorators and set hooks
-that will then be used in the rest of your application.
-
-Check out:
-
-* [The hitchhiker's guide to plugins](https://github.com/fastify/fastify/blob/master/docs/Plugins-Guide.md)
-* [Fastify decorators](https://www.fastify.io/docs/latest/Decorators/).
-* [Fastify lifecycle](https://www.fastify.io/docs/latest/Lifecycle/).
```

Comment:
The documentation of this README file should be put somewhere in the template.

Reply:
can we add documentation to this repo, and link to it instead?

Reply:
caught up on phone - @mcollina is fine with linking docs